jest.config.ts

```diff
@@ -5,17 +5,11 @@
 import type { Config } from 'jest';
 
 const config: Config = {
-    verbose: true,
-    preset: 'ts-jest',
-    testEnvironment: 'node',
-    transformIgnorePatterns: ['/dist/.+\\.js'],
-    // We need to explicity ignore the out directory for modules - otherwise we'll get duplicate vscode module,
-    // the TS version from the __mocks__ directory and the compiled js version from the out directory.
-    modulePathIgnorePatterns: ['out'],
-    // Specify jest to only run tests in jest folders.
-    // We also have to include the __mocks__ folder.  That folder must be next to node_modules so we can't move it,
-    // but if we specify roots, jest won't automatically pick it up.  So we have to specify it here.
-    roots: ['<rootDir>/test/unitTests', '<rootDir>/omnisharptest/omnisharpJestTests', '<rootDir>/__mocks__'],
+    projects: [
```

Comment:
we define separate jest 'projects' for our test runs.  this allows us to easily select which set to run and have different configurations for each.